schemas/1.0/general/provider.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "ministryofjustice/laa-criminal-legal-aid-schemas/main/schemas/1.0/general/provider.json",
+  "title": "Provider",
+  "description": "Holds common attributes of a provider",
+  "type": "object",
+  "properties": {
+    "office_code": { "type": "string" },
```

Comment:
Was it intentional to remove the minLength attr?

Reply:
Intentional. It was introduced originally when we didn't have a proper LAA Portal, and one mock user returned 0 office codes leaving this as blank.
As we don't have that problem anymore, I'd rather not have this arbitrary validation (in theory office codes will be 6 chars length but not sure if we want to have that constraint at all in the schema).